parquet-column/src/main/java/parquet/schema/Types.java

```diff
@@ -599,7 +988,7 @@ public static MessageTypeBuilder buildMessage() {
    * @return a {@link GroupBuilder}
    */
   public static GroupBuilder<GroupType> requiredGroup() {
-    return new GroupBuilder<GroupType>(GroupType.class)
+    return (GroupBuilder<GroupType>) new GroupBuilder<GroupType>(GroupType.class)
```

Comment:
This may be because I changed some of the types, so I recommend fixing the type stuff first and then seeing what is needed.